lib/modules/datasource/conda/index.spec.ts

```diff
@@ -1,13 +1,252 @@
+import { DateTime } from 'luxon';
 import { getPkgReleases } from '..';
 import { EXTERNAL_HOST_ERROR } from '../../../constants/error-messages';
 import { datasource, defaultRegistryUrl } from './common';
 import { CondaDatasource } from './index';
-import { Fixtures } from '~test/fixtures';
 import * as httpMock from '~test/http-mock';
 
 const packageName = 'main/pytest';
 const depUrl = `/${packageName}`;
 
+const anacondaApiRes = `
```

Comment:
you can use regular json object here for passing it to nock

please remove properties which we don't need for schema validation to reduce the size